pkgs/development/python-modules/marimo/default.nix

```diff
@@ -45,20 +49,19 @@ buildPythonPackage rec {
     itsdangerous
     jedi
     markdown
+    narwhals
     packaging
     psutil
     pygments
     pymdown-extensions
+    pyyaml
     ruff
     starlette
     tomlkit
     uvicorn
     websockets
-    pyyaml
   ];
 
-  nativeCheckInputs = [ pytestCheckHook ];
```

Comment:
If the tests are exclude then an option would be to use the source from GitHub. Persuade upstream to include the tests in PyPI releases usually takes some time.